importexport/musicxml/importmxmlpass2.cpp

```diff
@@ -6032,30 +6083,78 @@ void MusicXMLParserNotations::addToScore(ChordRest* const cr, Note* const note,
                                          TrillStack& trills, Tie*& tie)
       {
       addArpeggio(cr, _arpeggioType, _logger, &_e);
-      addAllArticulationsToChord(cr, _articulationSymbols);
       addBreath(cr, cr->tick(), _breath);
-      addFermata(cr, _fermataType, _fermataSymbol);
-      addStrongAccent(cr, _strongAccentType);
       addWavyLine(cr, Fraction::fromTicks(tick), _wavyLineNo, _wavyLineType, spanners, trills, _logger, &_e);
 
-      for (const auto& notation : _notations) {
-            if (notation.name() == "slur") {
-                  addSlur(notation, slurs, cr, tick, _logger, &_e);
+      for (const auto& annotation : _annotations) {
+            if (annotation.symId() != SymId::noSym) {
+                  QString direction = annotation.attribute("type");
```

Comment:
Why put the value of an attribute called type in a variable called direction instead of type ?